### PR TITLE
Supported tags based on metric name or group

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-811

**Description:**

1) Make group api running.
2) Test query api with multiple tags for particular group - working.
3) removed multiple dropdown values inside tags. 

 **Testing:**

`npm run test:unit`

 **Screenshots:**
Screenshot 2021-07-21 at 6 13 28 PM

(if applicable)

### CODE REVIEW

File: src/app/_shared/utils.ts [hunk 1]
```diff
@@ -57,9 +57,24 @@ function implementsObject<T>(obj: any, keys: (keyof T)[]): boolean {
     return true;
 }
 
+function filteredTags(tagObj) {
+    // Item that needs to be excluded from the obj
```
Comment (pras7430) [inline src/app/_shared/utils.ts:61]: we can use a single array with all excluded values

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { ActivatedRoute, Params, Router } from '@angular/router';
 import { Visualize } from '@minerva/_models/metrics'
 import { presetDates } from '@minerva/_features/visualize/components/timerange/timerange.component';
 import moment from 'moment';
+import { filteredTags } from '../../../../_shared/utils';
```
[the remark this answers is not in the document]
Comment (lovedeep92) [reply src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts:7]: yeah .. added short-cut path

File: src/app/_shared/utils.ts (hunk 1 above)
Comment (lovedeep92) [reply src/app/_shared/utils.ts:61]: done !!

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts [hunk 3]
```diff
@@ -109,6 +113,7 @@ export class VisualizePage {
     this.visualize.group = ["Select a Metric Group"];
   }
   ddTagInit() {
+    this.visualize.tags = [];
```
[the remark this answers is not in the document]
Comment (lovedeep92) [reply src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts:116]: yes .. done!

**Review by pras7430 — APPROVED** [review]
looks good to me

**Pull request merged**